packages/eds-core-react/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -542,6 +542,7 @@ function AutocompleteInner<T>(
           position: strategy,
           top: y ?? 0,
           left: x ?? 0,
+          zIndex: 1500,
```

Comment:
If it worked with `1400` should we not use that then? Autocomplete dropdown and Menu should have the same z-index

Reply:
yes good point